<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginform</class>
 <widget class="QWidget" name="loginform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>century gothic</family>
    <pointsize>-1</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;
	font-size: 20px;
}

#loginform{
	background-image: url(:/bg-01.jpg);
 	border: none;
}

QFrame{
	background: #333;
 	border-radius: 10px;
}

#login_btn{
	background: red;
	color: white;
}

#login_btn:hover{
	background-color: #00FFFF;
	color: red;
	border: none;
}

QLineEdit{
	background: transparent;
	border: none;
	color: #717072;
	border-bottom: 1px solid #707172;
	border-radius: 0px;
}

#Back_btn{
	border: none;
	background: transparent;
	color: white;
}

#label_2{
	color: #808080;
	font-size:12px;
}

#Register{
	color: white;
	font-size: 14px;
	background: #333;
	border: none;
}

#forgot{
	font-size: 12px;
	color: white;
	background: transparent;
	border: none;
}

#img{
	background: transparent;
}

#show_password{
	background: transparent;
	border: none;
}

#warning_username, #warning_password{
	font-size: 9px;
	padding-left: 5px;
	color: red;
	font-weight: bold;
	text-align: left;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>300</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
text-align: center;
background: transparent;</string>
    </property>
    <property name="text">
     <string>LOGIN HERE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string extracomment="Username"/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">Username</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5px;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="forgot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Forgot password?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Have not account yet?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Register">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_password">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource/user_icon.qrc">
      <normaloff>:/hide-password.svg</normaloff>:/hide-password.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="warning_username">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="warning_password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-radius: 30px;
background: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resource/user_icon.qrc">
     <normaloff>:/user-regular.svg</normaloff>:/user-regular.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resource/user_icon.qrc">
     <normaloff>:/arrow-left-solid.svg</normaloff>:/arrow-left-solid.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-image: url(:/bg-01.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>pushButton</zorder>
  <zorder>Back_btn</zorder>
 </widget>
 <resources>
  <include location="Resource/user_icon.qrc"/>
 </resources>
 <connections/>
</ui>
